File: LeetCode/69.Sqrt_X/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    int t1 = 4;
    int t2 = 8;
    if (s.mySqrt(t1) == 2) { cout << "Pass" << endl;}
    else { cout << "Failed 1 - t1" << endl;}
    if (s.mySqrt(t2) == 2) { cout << "Pass" << endl;}
    else { cout << "Failed 2 - t2" << endl;}
    return 0;
}

File: LeetCode/69.Sqrt_X/main.py
#!/usr/bin/python

from solution import Solution

# Unit Test
t1 = 4
t2 = 8
s = Solution()
if (s.mySqrt(t1)==2):
    print("Pass")
else:
    print("Test 1 Failed")
if (s.mySqrt(t2) == 2):
    print("Pass")
else:
    print("Test 2 Failed")